cmd/helm/lint.go

```diff
@@ -47,10 +45,15 @@ type lintCmd struct {
 	valueFiles valueFiles
 	values     []string
 	sValues    []string
+	fValues    []string
 	namespace  string
 	strict     bool
 	paths      []string
 	out        io.Writer
+
+	certFile string
+	keyFile  string
+	caFile   string
```

Comment:
Why were TLS options added to Lint, which is explicitly forbidden from making server connections?

Reply:
True! I don't remember why I did it but I'll remove it asap. Thx.

Reply:
Turned out it was due to my misunderstanding. Reverted the tls options and the relevant refactoring altogether - It should be good to go now!

The longer explanation follows.

Firstly, I removed the `vals` func dedicated to the `lint` command and migrated it to reuse another `vals` func used across other commands like `upgrade`.

While migrating,  I believe that I thought it would be better to add support for tls opts for the `vals` func, for use when downloading remote value files. And it resulted in adding those flags. But as you pointed out, it conflicts with the design of the `lint` command, that is forbidden from making server connections. I reverted those changes and it should be good now :)